jans-linux-setup/jans_setup/templates/jans-auth/role-scope-mappings.json

```diff
@@ -0,0 +1,380 @@
+        "https://jans.io/oauth/config/fido2.readonly",
+        "https://jans.io/oauth/config/fido2.write",
+        "https://jans.io/oauth/config/cache.readonly",
+        "https://jans.io/oauth/config/cache.write",
+        "https://jans.io/oauth/config/database/couchbase.readonly",
+        "https://jans.io/oauth/config/database/couchbase.write",
+        "https://jans.io/oauth/config/database/sql.readonly",
+        "https://jans.io/oauth/config/database/sql.write",
+        "readonly",
+        "https://jans.io/oauth/config/stats.readonly",
+        "jans_stat"
+      ]
+    },
+    {
+      "role": "api-manager",
+      "permissions": [
+        "https://jans.io/oauth/config/attributes.readonly",
+        "https://jans.io/oauth/config/attributes.write",
+        "https://jans.io/oauth/config/attributes.delete",
+        "https://jans.io/oauth/config/acrs.readonly",
+        "https://jans.io/oauth/config/acrs.write",
+        "https://jans.io/oauth/config/acrs.delete",
+        "https://jans.io/oauth/config/scopes.readonly",
+        "https://jans.io/oauth/config/scopes.write",
+        "https://jans.io/oauth/config/scopes.delete",
+        "https://jans.io/oauth/config/scripts.readonly",
+        "https://jans.io/oauth/config/scripts.write",
+        "https://jans.io/oauth/config/scripts.delete",
+        "https://jans.io/oauth/config/openid/clients.readonly",
+        "https://jans.io/oauth/config/openid/clients.write",
+        "https://jans.io/oauth/config/openid/clients.delete",
+        "https://jans.io/oauth/config/smtp.readonly",
+        "https://jans.io/oauth/config/smtp.write",
+        "https://jans.io/oauth/config/smtp.delete",
+        "https://jans.io/oauth/config/logging.readonly",
+        "https://jans.io/oauth/config/logging.write",
+        "https://jans.io/oauth/config/uma/resources.readonly",
+        "https://jans.io/oauth/config/uma/resources.write",
+        "https://jans.io/oauth/config/uma/resources.delete",
+        "https://jans.io/oauth/config/database/ldap.readonly",
+        "https://jans.io/oauth/config/database/ldap.write",
+        "https://jans.io/oauth/config/database/ldap.delete",
+        "https://jans.io/oauth/config/jwks.readonly",
+        "https://jans.io/oauth/config/jwks.write",
+        "https://jans.io/oauth/config/fido2.readonly",
+        "https://jans.io/oauth/config/fido2.write",
+        "https://jans.io/oauth/config/cache.readonly",
+        "https://jans.io/oauth/config/cache.write",
+        "https://jans.io/oauth/config/database/couchbase.readonly",
+        "https://jans.io/oauth/config/database/couchbase.write",
+        "https://jans.io/oauth/config/database/sql.readonly",
+        "https://jans.io/oauth/config/database/sql.write",
+        "readonly",
+        "https://jans.io/oauth/config/stats.readonly",
+        "jans_stat"
+      ]
+    },
+    {
+      "role": "api-admin",
+      "permissions": [
+        "https://jans.io/oauth/config/attributes.readonly",
```

Comment:
Add this `permission` to `api-admin` role mapping
https://jans.io/oauth/jans-auth-server/config/properties.readonly

Reply:
> Please add below `permission` to `api-admin` role mapping https://jans.io/oauth/jans-auth-server/config/properties.readonly

Don't worry about it. All scopes from config-api swagger yaml file is added to `api-admin`